tempto-core/src/main/java/com/teradata/tempto/internal/listeners/ProgressLoggingListener.java

```diff
@@ -87,8 +86,15 @@ public void onTestSkipped(ITestResult testCase)
 
     private void logTestEnd(String outcome)
     {
-        long executionTime = System.currentTimeMillis() - testStartTime;
-        LOGGER.info("{}     /    took {}", outcome, DurationFormatUtils.formatDuration(executionTime, "m' minutes and 's' seconds'"));
+        if (LOGGER.isInfoEnabled()) {
+            long executionTime = System.currentTimeMillis() - testStartTime;
+            if (executionTime > 1000) {
```

Comment:
It is meaningless to display to most of the tests that they last 0 minutes and 0 second.